src/components/global/header/headerNotificationPanel.ts

```diff
@@ -35,6 +57,13 @@ export class HeaderNotificationPanel extends LitElement {
             <h1 class="panel-header-text">${this.panelTitle}</h1>
           </div>
           <div class="panel-left-slot">
+            <kd-button
+              kind="tertiary"
+              @click=${(e: Event) => this._toggleMarkAllReadUnread(e)}
+              >${this.markAllRead
+                ? this.textStrings.markAllUnread
+                : this.textStrings.markAllRead}</kd-button
+            >
```

Comment:
I think we should externalize this, and place it in the menu-slot with the settings icon.